worlds/overcooked2/__init__.py

```diff
@@ -508,6 +508,8 @@ def kevin_level_to_keyholder_level_id(level_id: int) -> Optional[int]:
             "SaveFolderName": mod_name,
             "CustomOrderTimeoutPenalty": 10,
             "LevelForceHide": [37, 38, 39, 40, 41, 42, 43, 44],
+            "LocalDeathLink": self.options["DeathLink"] != DeathLinkMode.disabled.value,
```

Comment:
.value should not be needed and in fact the object has additional safety checks in place for type and content safety

Reply:
another thing to note here is that you can do direct comparisons with the options themselves, so you could do `self.options["DeathLink"] != DeathLink.option_disabled` instead of implementing enums